examples/deployment/docker-compose.yml

```diff
@@ -25,4 +25,15 @@ services:
     environment:
       - DB_USER=root
       - DB_PASSWORD=$DB_PASSWORD
-      - DB_HOST=mysql:3306
+  trillian-log-signer:
+    build:
+      context: ../..
+      dockerfile: examples/deployment/docker/log_signer/Dockerfile
+    restart: always # retry while mysql is starting up
+    ports:
+      - "8092:8091"
+    depends_on:
+      - mysql
+    environment:
+      - DB_USER=root
+      - DB_PASSWORD=$DB_PASSWORD
```

Comment:
Are you sure this environment variable is being picked up in the way you expect? 
I seem to recall that env variable replacement was a missing feature.

Reply:
I think it does. I was able to run this locally. I also tried to manually connect to the db within the mysql container, and it indeed had the correct user/password.